src/fastq_summ/Makefile

```diff
@@ -5,12 +5,13 @@ CFLAGS = -g -Wall
 BUILDDIR = build
 installdir = .
 
-INCLUDES= -I.
-SAMTOOLS_LOC= /software/solexa/pkg/samtools/samtools-0.1.18
-LIBS= -lz -lm
+SAMTOOLS_LOC= /software/solexa/pkg/samtools/samtools-1.2
+HTSLIB_LOC= /software/solexa/pkg/htslib/htslib-1.2.1
+INCLUDES= -I. -I$(SAMTOOLS_LOC)/include -I$(HTSLIB_LOC)/include
+LIBS= -lz -lm -Wl,-rpath=$(HTSLIB_LOC)
```

Comment:
Think rpath option is missing suffix: should be
`LIBS= -lz -lm -Wl,-rpath=$(HTSLIB_LOC)/lib`